<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yonje_dialog</class>
 <widget class="QDialog" name="Yonje_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="capacity_yonje_l">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tedad_barayekashtspinBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lvl_fyonje_l">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upgrade_zaminy_butt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bardasht_butt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>scythe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="nzirkesht_l">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lvl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Under crop cultivation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kashtg_butt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Gardening</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plow_butt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Plow</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
